.github/workflows/pr.yaml

```diff
@@ -141,8 +141,10 @@ jobs:
       script: ci/test_wheel_dask_cudf.sh
   devcontainer:
     secrets: inherit
-    uses: rapidsai/shared-action-workflows/.github/workflows/build-in-devcontainer.yaml@branch-24.04
+    uses: rapidsai/shared-workflows/.github/workflows/build-in-devcontainer.yaml@fix/devcontainer-json-location
```

Comment:
Can we merge that `shared-workflows` PR into `branch-24.04` before merging this PR? Or is there a dependency between the merge order?

Reply:
While I think it should be fine if we merge in any order, the safe thing to do is to merge this before [the shared-workflows PR](https://github.com/rapidsai/shared-workflows/pull/175).

Reply:
The `build-in-devcontainer.yaml` was hard-coded to CUDA 12.0 and the new workflow makes that configurable.